packages/fxa-payments-server/src/routes/Product/SubscriptionRedirect/index.tsx

```diff
@@ -1,33 +1,44 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Plan } from '../../../store/types';
 
 import './index.scss';
 
+// Mapping of product IDs to post-subscription redirect URLs
+// TODO: Move / merge in some kind of configuration variable?
+type productRedirectURLsType = { [productID: string]: string };
+const productRedirectURLs: productRedirectURLsType = {
+  '123doneProProduct': 'http://127.0.0.1:8080/',
+  'prod_Ex9Z1q5yVydhyk': 'https://123done-latest.dev.lcip.org/',
+  'prod_FUUNYnlDso7FeB': 'https://123done-stage.dev.lcip.org',
```

Comment:
It really seems like these URLs live in a config file, not directly in the code like this. How about just moving these URLs out to a config.js file, following the approach in auth server config?